<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationPanelWidget</class>
 <widget class="QWidget" name="StationPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="stationTabWidget">
     <property name="currentIndex">
      <number>4</number>
     </property>
     <widget class="QWidget" name="recentTab">
      <attribute name="title">
       <string>Recent</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="recentList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tagsTab">
      <attribute name="title">
       <string>Tags</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="tagsList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="friendsTab">
      <attribute name="title">
       <string>Friends</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="friendsList" native="true"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLineEdit" name="friendsFilter"/>
         </item>
         <item>
          <widget class="QPushButton" name="friendsSort">
           <property name="text">
            <string>sort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="nieghboursTab">
      <attribute name="title">
       <string>Neighbours</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="neighboursList" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="artistTab">
      <attribute name="title">
       <string>Artists</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="StationWidget" name="artistsList" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header>../widgets/StationWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
